### DISCUSSION TITLE
Created a video explaining how the code for ChatHN

### DISCUSSION THREAD

**Discussion by elie222** [steven-tey/chathn#4] [Show and tell]
Great project. Always a good chance to learn about new packages and what's possible with Vercel.
Put together two videos explaining this repo:

1. https://youtu.be/hcNJaH4EHNY
2. https://youtu.be/QY7U5GxanEU

**Comment by steven-tey** [comment]
Thank you for making these videos – love them!!

**Reply by elie222** [reply]
Might do one on dub.sh next week :)
